scalikejdbc-core/src/main/scala/scalikejdbc/TxBoundary.scala

```diff
@@ -41,10 +41,15 @@ object TxBoundary {
     }
   }
 
+  private[scalikejdbc] sealed abstract class TxBoundaryMissingImplicits {
+    implicit def `"To activate TxBoundary.Future, scala.concurrent.ExecutionContext value in implicit scope is required here."`[A]: TxBoundary[A] = sys.error("use internalTxBoundary!")
```

Comment:
mm, It means 'this Txboundary is internal implementation'.
Do you think that I will change this message to 'Don't use this directly' ?